src/go/rpk/pkg/system/syslog/syslog.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// +build linux
+
+package syslog
+
+import (
+	"fmt"
+	"syscall"
+)
+
+const (
+	// Taken from `man syslog`
+
+	// Read  all messages remaining in the ring buffer, placing them in the buffer pointed
+	// to by bufp.  The call reads the last len bytes from  the  log  buffer  (nondestruc‐
+	// tively),  but  will  not  read more than was written into the buffer since the last
+	// "clear ring buffer" command.  The call returns the number of bytes read.
+	syslog_action_read_all int = 3
```

Comment:
I think since this is a c&p of the man docs, maybe the doc comment can be left out, and replaced with `See man 2 syslog`.  It makes more sense in context in the man page itself, since `len` is underlined and clearly refers to the arg to the klogctl function.

Same comment for size_buffer below.

I can't quite tell whether snake_case is good here... definitely violates Go naming guidelines, but it does match directly to the man page. I think if we're going per the man page it may make sense to have this be uppercased, otherwise maybe nix the underscore?

Reply:
Actually, these constants can be removed and replaced with the unix package equivalents:
unix.SYSLOG_ACTION_READ_ALL
unix.SYSLOG_ACTION_SIZE_BUFFER

Reply:
Ah, check it out! Nice find, thanks.

Reply:
Fixed.